src/cluster.ts

```diff
@@ -321,6 +429,116 @@ Please use the format 'my-cluster' or '${instance.name}/clusters/my-cluster'.`);
+  getBackups(options: GetBackupsOptions, callback: GetBackupsCallback): void;
+  getBackups(callback: GetBackupsCallback): void;
+  /**
+   * Get Cloud Bigtable Backup instances within this cluster. This returns both
+   * completed and pending backups.
+   *
+   * @param {GetBackupsOptions | GetBackupsCallback} [optionsOrCallback]
+   * @param {GetBackupsResponse} [cb]
+   * @return {void | Promise<ListBackupsResponse>}
+   */
+  getBackups(
+    optionsOrCallback?: GetBackupsOptions | GetBackupsCallback,
+    cb?: GetBackupsCallback
+  ): void | Promise<GetBackupsResponse> {
+    const options =
+      typeof optionsOrCallback === 'object' ? optionsOrCallback : {};
+    const callback =
+      typeof optionsOrCallback === 'function' ? optionsOrCallback : cb!;
+
+    const reqOpts: google.bigtable.admin.v2.IListBackupsRequest = {
+      ...options,
+      parent: this.name,
+    };
+
+    delete (reqOpts as GetBackupsOptions).gaxOptions;
+
+    this.bigtable.request<google.bigtable.admin.v2.IBackup[]>(
+      {
+        client: 'BigtableTableAdminClient',
+        method: 'listBackups',
+        reqOpts,
+        gaxOpts: options.gaxOptions,
+      },
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      (err, ...resp: any[]) => {
+        let backups: Backup[] = [];
+
+        if (resp[0]) {
+          backups = resp[0].map((backup: IBackup) => {
+            const backupInstance = this.backup(backup.name!.split('/').pop()!);
+            backupInstance.metadata = backup;
+            return backupInstance;
+          });
+        }
+        const nextQuery = resp[1]! ? Object.assign({}, options, resp[1]) : null;
+        const apiResp: google.bigtable.admin.v2.IListBackupsResponse = resp[2];
+
+        callback(err, backups, nextQuery, apiResp);
+      }
+    );
+  }
+
+  /**
+   * Lists Cloud Bigtable backups within this cluster. Provides both
+   * completed and pending backups as a readable object stream.
+   *
+   * @param {GetBackupsOptions} [options] Configuration object. See
+   *     {@link Cluster#getBackups} for a complete list of options.
+   * @returns {ReadableStream<Backup>}
+   *
+   * @example
+   * const {Bigtable} = require('@google-cloud/bigtable');
+   * const bigtable = new Bigtable();
+   * const instance = bigtable.instance('my-instance');
+   * const cluster = instance.cluster('my-cluster');
+   *
+   * cluster.getBackupsStream()
+   *   .on('error', console.error)
+   *   .on('data', function(backup) {
+   *     // backup is a Backup object.
+   *   })
+   *   .on('end', () => {
+   *     // All backups retrieved.
+   *   });
+   *
+   * //-
+   * // If you anticipate many results, you can end a stream early to prevent
+   * // unnecessary processing and API requests.
+   * //-
+   * cluster.getBackupsStream()
+   *   .on('data', function(backup) {
+   *     this.end();
+   *   });
+   */
+  getBackupsStream(options: GetBackupsOptions): NodeJS.ReadableStream {
```

Comment:
For this method to work `listBackupsStream: true` needs to be added to [`gapicStreamingMethods`](https://github.com/googleapis/nodejs-bigtable/blob/2a199aae5646fe4e19836e13a61fcb83b527e4c5/src/index.ts#L753)